packages/foreman/rubygem-jquery-ui-rails/rubygem-jquery-ui-rails.spec

```diff
@@ -1,75 +1,95 @@
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
-# Generated from jquery-ui-rails-4.0.2.gem by gem2rpm -*- rpm-spec -*-
 %global gem_name jquery-ui-rails
 
-Summary: jQuery UI packaged for the Rails asset pipeline
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 4.1.2
-Release: 9%{?dist}
+Version: 6.0.1
+Release: 1%{?dist}
+Summary: jQuery UI packaged for the Rails asset pipeline
 Group: Development/Languages
 License: MIT
 URL: https://github.com/joliss/jquery-ui-rails
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby(rubygems) >= 1.3.6
 Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix}rubygem(railties) >= 3.1.0
+Requires: %{?scl_prefix_ruby}ruby(rubygems) >= 1.3.6
+Requires: %{?scl_prefix}rubygem(railties) >= 3.2.16
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.6
 BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.6
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
```

Comment:
Is the `Obsolete` still needed?

Reply:
Not really, we've started to drop them in other packages as well.